cypress/integration/ProcessorTest.ts

```diff
@@ -310,43 +327,38 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
         assertSourceProcessorDetails(processorName, source, filters);
       });
 
-      onlyOn(bugPresent, () => {
-        it("Add filter row", () => {
-          if (bugPresent) {
-            cy.log(bugPresent);
-            return;
-          }
+      it("Add filter row", () => {
+        filters[0].Key = "data.surname";
```

Comment:
@jomarko There could be a problem

Reply:
Do you think that there could be: `filters[1]["Key"]`?

Reply:
I am not `TS` expert programmer, could you please clarify your concern?

is the problem index I used `0` vs `1`  , or `.Key` vs `["Key"]`?

Reply:
@jomarko  I mean that `.Key` might caused that problem. I am not TS expert too but based on my Java and JS experience:
1.`.Key` reference the property of an object which is saved in the `fitlers` array under index `0`
2. `["Key"]` reference the item in an array/map structure which key is `"Key"` this array/map is saved in the `filters` array under index '1'.

Reply:
@jomarko the error message is <failure message="Cannot set properties of undefined (setting 'Key')" type="TypeError">

Reply:
ok, pushing separate commit with `["xxx"]` syntax